README.md

```
@@ -1,30 +1,40 @@
+<a href="https://crates.io/crates/hyperfuel-client">
+<img src="https://img.shields.io/crates/v/hyperfuel-client.svg?style=flat-square"
+    alt="Crates.io version" />
+</a>
+
+Rust crate for [Envio's](https://envio.dev/) HyperFuel client.
 
 `$ cargo add hyperfuel-client`
 
+[Documentation Page](https://docs.envio.dev/docs/hypersync-clients)
+
 ### Dependencies
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)